test/db/formats/mz/cblp0

```diff
@@ -0,0 +1,32 @@
+NAME=MZ: cblp0.exe - open
+FILE=bins/mz/cblp0.exe
+ARGS=-A
+CMDS=q!
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=MZ: cblp0.exe - entrypoint
+FILE=bins/mz/cblp0.exe
+CMDS=s
+EXPECT=<<EOF
+0x0
+EOF
+RUN
+
+NAME=MZ: cblp0.exe - pi 1
```

Comment:
you can run pio 1 and iS* in the same test. the more tests we have the slower the CI runs, so its better to squash some tests to work together in case those cant be side-effected.